### PR TITLE
fix(infra.ci): add command for new node agent

### PR DESCRIPTION
**Pull request by smerle33**
add a command to the node.

### CODE REVIEW

File: config/ext_jenkins-infra.yaml [hunk 1]
```diff
@@ -233,7 +234,7 @@ controller:
                             value: "~/.asdf/shims:~/.asdf/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin"
                         - envVar:
                             key: "ARTIFACT_CACHING_PROXY_PROVIDER"
-                            value: "do"
+                            value: "azure"
```
Comment (lemeurherve) [inline config/ext_jenkins-infra.yaml:237]: Why this change?

Comment (dduportal) [reply config/ext_jenkins-infra.yaml:237]: yep: the containers issued for infra.ci are all in the same cluster (temp-privatek8s)  e.g. inside Azure.

The initial value here comes from a copy and paste from ci.jenkins.io.

**Review by dduportal — APPROVED** [review]
